DirectX11/util.h

```diff
@@ -0,0 +1,308 @@
+#pragma once
```

Comment:
I had gone out of my way to make this util.h work for DX9+DX11 and as much as possible I don't want yet more copies of the same code (we already have too much of that with DX10 being a copy+paste of DX11 and Dave's DX10 port being the same), so please move it back.